tinyagentos/routes/cluster_migrate.py

```
@@ -0,0 +1,209 @@
+
+    try:
+        result = await migrate_service(
+            body.app_id,
+            body.target_remote,
+            install_config=manifest.install,
+            state_paths=state_paths,
+            service_name=service_name,
+            keep_source=body.keep_source,
+            source_remote=body.source_remote,
+        )
+    except Exception as exc:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)